<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletWidget</class>
 <widget class="QWidget" name="WalletWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="entryList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="addEntryBtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="list-add">
             <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeEntryBtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="list-remove">
             <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="contentPages">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="emptyPage"/>
     <widget class="QWidget" name="contentWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="showPasswordsCheck">
         <property name="text">
          <string>Show passwords</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="entryPaneContent">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>529</width>
            <height>683</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="entryPaneLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="entryFields" native="true">
             <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1" columnminimumwidth="10,0,0">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="1">
               <widget class="QLabel" name="usernameLabel">
                <property name="text">
                 <string>Username</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="passwordLabel">
                <property name="text">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="numCol">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="addPasswordBtn">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="list-add">
               <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addEntryBtn</sender>
   <signal>clicked()</signal>
   <receiver>WalletWidget</receiver>
   <slot>onAddEntryBtnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>703</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeEntryBtn</sender>
   <signal>clicked()</signal>
   <receiver>WalletWidget</receiver>
   <slot>onRemoveEntryBtnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>703</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPasswordBtn</sender>
   <signal>clicked()</signal>
   <receiver>WalletWidget</receiver>
   <slot>onAddPasswordBtnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showPasswordsCheck</sender>
   <signal>clicked()</signal>
   <receiver>WalletWidget</receiver>
   <slot>onShowPasswordsPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddEntryBtnPressed()</slot>
  <slot>onRemoveEntryBtnPressed()</slot>
  <slot>onAddPasswordBtnPressed()</slot>
  <slot>onShowPasswordsPressed()</slot>
 </slots>
</ui>
